rtrlib/rtr/packets.c

```diff
@@ -47,6 +48,28 @@ enum pdu_type {
     ERROR = 10
 };
 
+// These modes let the user configure how
+// received intervals should be handled.
+enum interval_mode {
```

Comment:
this enum might be needed for a future config API call, so might be good to move it into a header file.
Further, please add doxygen conform documentation for this enum then.